### DISCUSSION TITLE
dfir-iris

### DISCUSSION THREAD

**Discussion by mysther** [selfhst/icons#157] [New Icon Request]
Hello,

Iris is a Digital Forensics and Incident Response opensource platform.
It's a sub-field of cybersecurity that focuses on identifying, remediating, and investigating cyber security incidents.

Didn't find any SVG (sorry), but there is their 
ICO : https://github.com/dfir-iris/iris-web/blob/master/source/app/static/assets/img/logo.ico
More choise https://github.com/dfir-iris/iris-web/tree/master/source/app/static/assets/img

Thanks for your project,

**Comment by shollyethan** [comment]
No worries - the logo consists of simple shapes, so I was able to easily recreate it to get it into the SVG format. It's been added to the repo (the project name is IRIS per its repo, but the actual URL reference in this collection is dfir-iris given 'iris' is a fairly generic name that may conflict with future icons).
